### ISSUE TITLE
Search Results: Insufficient Color Contrast (2.9:1) For Description Heading

### ISSUE THREAD

**Issue by jscholes** [benetech/ImageShare#60]
The "Description" heading needs to have at least a 3:1 color contrast ratio.  Consider changing the color of the text to `#0C4471` to match the other headings.

![Screenshot of the "Description" heading appearing in light red directly below the blue Heart Chambers heading.](https://user-images.githubusercontent.com/6454755/80004176-a1e7dd00-8487-11ea-87d1-ab3dd105c150.png)

**Comment by clapierre** [comment]
I would like to have a Contrast ratio of at least 4.5 to 1 and ideally 7 to 1 for WCAG-AAA compliance.  Now Large bolded text wouldn't need to 4.5:1 to be AAA compliant which would be a good goal to reach if we can.  I am fine to adjust the colors to meet this.    Provide me with the color options to meet this and I would think we are golden.

**Comment by sinabahram** [comment]
Can either @jkva  or @vincent-young-1701  please confirm this is true in current implementation and then close this issue if true?

**Comment by vincent-young-1701** [comment]
This has been taken care of in the new.
